### DISCUSSION TITLE
Mandatory "spec" field in custom resources

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#546] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

13

### Description

Hi!

From [this documentation](https://docs.mia-platform.eu/docs/console/design-your-projects/custom-resources#update-a-custom-resource) I noticed that the "spec" field is required for the custom resources.

However, Kuberntes resources like `Role` or `RoleBinding` do not expect the "spec" field.
Is there a way to overcome this limitation?

Thanks.

**Comment by Danielecina** [comment]
Hi @Lucascanna, I checked the definition schema of a custom resource and I can assure you that it is not required on the marketplace side. I tried unfortunately to save a custom resource without spec in the Frontend and it gives me errors when saving, so I would say it is something unexpected of this beta version. 

In the coming weeks we will take that part of the product in hand and work on it with this issue in mind

**Reply by Danielecina** [reply, marked as answer]
Comparing for a second I learned that the spec is wanted as the required value for now. 
So let's update the issue if there are any developments.
It would be very convenient if you have other useful usage examples
Thanks
